stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -362,6 +362,8 @@ extension DistributedActor {
   /// state.
   ///
   /// When the actor is remote, the closure won't be executed and this function will return nil.
+  @inlinable
+  @_alwaysEmitIntoClient
```

Comment:
AEIC implies `@inlinable`.

Reply:
Maybe so... All sites where I see typed throws adopted spell it out doubly-so like this... I'll follow the pattern and we can do repo wide cleanup pass separately perhaps